polyply/data/2016H66/citation.bib

```diff
@@ -0,0 +1,18 @@
+@article{2016H66,
+  title={A GROMOS-compatible force field for small organic molecules in the condensed phase: The 2016H66 parameter set},
+  author={Horta, Bruno AC and Merz, Pascal T and Fuchs, Patrick FJ and Dolenc, Jozica and Riniker, Sereina and Hünenberger, Philippe H},
+  journal={Journal of chemical theory and computation},
+  volume={12},
+  number={8},
+  pages={3825--3850},
+  year={2016},
+  publisher={ACS Publications}
+}
+
+@article{polyply,
+  title={Polyply; a python suite for facilitating simulations of (bio-) macromolecules and nanomaterials},
+  author={Grunewald, Fabian and Alessandri, Riccardo and Kroon, Peter C and Monticelli, Luca and Souza, Paulo CT and Marrink, Siewert J},
+  journal={Nature Communications},
+  doi={10.1038/s41467-021-27627-4},
```

Comment:
following https://www.nature.com/articles/s41467-021-27627-4, you can already add: 
```
volume={13},
pages={68},
```